mlibrary/catalog-browse
# [SEARCH-1542] Browses starting with unexpected characters/string result in a NoMethodError page

# Overview
The `NoMethodError` error is a bit much for users, so we are creating an error message to appear under the search box `input` that informs the user that the call number value in invalid.

This pull request resolves [JIRA-1542](https://tools.lib.umich.edu/jira/browse/JIRA-1542).

## Anything else?
`./public/browse.css` has been getting long, so the stylesheets have been broken up into more bite-sized files. Then detailed commenting has been added to each one for documentation.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Enter an invalid callnumber (e.g. `ML`) to see the error message appear.
